src/foam/nanos/notification/email/GroupEmailTemplateService.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.notification.email',
+  name: 'GroupEmailTemplateService',
+
+  documentation: 'Used in conjuction with ChainedPropertyService',
```

Comment:
Remove mention of this and describe what this class does instead:
Fills unset properties on an email with values from the group.

Reply:
👍 Thx for the suggestion 😉